plugin.js

```diff
@@ -70,31 +70,30 @@ function onReqHandlerWrapper (fastify, hook) {
 }
 
 function setCookies (reply) {
-  let setCookie = reply.getHeader('Set-Cookie')
-  const setCookieIsUndefined = setCookie === undefined
+  const setCookieHeader = reply.getHeader('Set-Cookie')
+  let setCookie
```

Comment:
If we are refactoring for clarity, I believe this will make things even clearer:
```suggestion
  const setCookieHeaderValue = reply.getHeader('Set-Cookie')
  let cookieValue
```

Reply:
Done